### ISSUE TITLE
Screen reader doesn't announce the correct value for the slider

### ISSUE THREAD

**Issue by EddyNordica** [EddyNordica/rubber-band-counter#1]
When using Narrator on Windows, the reported value of the slider is incorrect when it's set to 1.

**Expected:** The reported value is 1.
**Actual:** The reported value is 0.
